buildSrc/src/main/java/com/r3/csde/VNode.java

```diff
@@ -0,0 +1,26 @@
+package com.r3.csde;
+
+
+/**
+ * This class is a representation of a Vnode used to express the vNodes required on the network.
+ */
+
+public class VNode{
+    private String x500Name;
+    private String cpi;
+
+    private String serviceX500Name;
+
+    public VNode() {
```

Comment:
for such a c'tor in Java can't you just not write it (This is the case for C#/C++ and probably Kotlin too)

Reply:
I don't believe so, I think you need it for jackson serialisation. But I will double check.

Reply:
okay, using a no constructor args is one of the ways jackson can deserialize. You can set up other constructors with annotations, but as this way works and is pretty straight forward I'd like to stick to it, given the time constraints we are under.